docs/tutorials/api/datasets.ipynb

```diff
@@ -0,0 +1,877 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Datasets - Reduced data, IRFs, models \n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "`gammapy.datasets` are a crucial part of the gammapy API. `datasets` constitute `DL4` data - binned counts, IRFs, models and the associated likelihoods. `Datasets` from the end product of the `makers` stage, see [makers notebook](makers.ipynb), and are passed on to the `Fit` or estimator classes for modelling and fitting purposes.\n",
```

Comment:
About "`Datasets` from the end product of the `makers` stage", is it not rather:
"`Datasets` are the products of the Data Reduction stage, with the use of `makers`, [...]"

Reply:
Thanks @bkhelifi ! I meant `form` not `from` :-/